slackbot-guysgals-reasoning.md

```diff
@@ -0,0 +1,34 @@
+# Why the Auto Response?
+
+The ZATech community has some people who feel excluded by certain words. In particular there was a point raised in #diversity about the word `guys`.
```

Comment:
I don't think it's necessary or useful to say "a lot of people who aren't men".
Just saying "a lot of people" sounds fine.